src/placement/tests/place_obj_common.c

```diff
@@ -396,16 +396,14 @@ plt_set_domain_status(uint32_t id, int status, uint32_t *ver,
 	rc = pool_map_find_domain(po_map, level, id, &domain);
 	D_ASSERT(rc == 1);
 
-	int i;
-
-	for (i = 0; i < domain->do_child_nr; i++) {
+	for (int i = 0; i < domain->do_child_nr; i++) {
```

Comment:
This function however does require changes so I've scoped the variable properly.